samples/msal-browser-samples/VanillaJSTestApp2.0/app/customizable-e2e-test/test/browser.spec.ts

```diff
@@ -20,8 +20,7 @@ async function enterCredentials(page: puppeteer.Page, screenshot: Screenshot): P
     await screenshot.takeScreenshot(page, "loginPage");
     await page.type("#i0116", username);
     await page.click("#idSIButton9");
-    await page.waitForNavigation({ waitUntil: "networkidle0"});
-    await page.waitForSelector("#i0118");
+    await page.waitForSelector("#idA_PWD_ForgotPassword");
```

Comment:
We don't need navigation anymore?

Reply:
The tests seem to run faster in this environment than on ADO so they were breaking because it was trying to enter the password on the username page. The new check is sufficient as it will wait until it finds the ForgotPassword button before attempting to enter a password. This was the only distinguishing element I could find between the username page and the password page, but should hopefully make the tests slightly more reliable.